app/controllers/concerns/order_stock_check.rb

```diff
@@ -25,7 +25,7 @@ def sufficient_stock?
   end
 
   def reset_order_to_cart
-    return if Flipper.enabled? :split_checkout
+    return if Flipper.enabled? :split_checkout, spree_current_user
```

Comment:
I'm wondering if we should use a helper and pass the user automatically.

https://github.com/openfoodfoundation/openfoodnetwork/blob/205fb630cb074c349b3cd3131593ead360ea5614/app/helpers/application_helper.rb#L26-L28

Shall we change the default and use that?
```rb
 def feature?(feature, user = spree_current_user) 
   OpenFoodNetwork::FeatureToggle.enabled?(feature, user) 
 end
```

Easier to write, less room for error and we stay open to changing the implementation in the future.

Reply:
Hmmm... there's various contexts where we can't really use the helper though, like models and route conditions. So we'd have to mix using `feature? :split_checkout` in some places and `Flipper.enabled? :split_checkout` in other places.

Reply:
I'm not sure `OpenFoodNetwork::FeatureToggle` really serves any purpose now..?

Reply:
You are right, our FeatureToggle class doesn't do much now, just an abstraction. But we could use it to enforce the user parameter. That could help to remember it.